### PR TITLE
Add hover effect for cells in segmentations

### PR DESCRIPTION
**Pull request by philippotto**
### Mailable description of changes:
 - When hovering over cells in segmentations, the cell under the mouse cursor will be highlighted (in all viewports). This makes it very easy to clarify which cells belong together and which don't.

### Steps to test:
- Hover over cells (test different brushing tools)

### Issues:
- fixes #123

------
- [x] Ready for review

### CODE REVIEW

**Comment by philippotto** [comment]
Video of the feature: http://sendvid.com/c36nukar?secret=98c332db-ae0a-41e8-b228-604c7d16afe4

**Comment by fm3** [comment]
@philippotto i added the default config value, does that work for you?

File: app/assets/javascripts/oxalis/controller/combinations/volumetracing_plane_controller.js [hunk 1]
```diff
@@ -204,11 +204,10 @@ class VolumeTracingPlaneController extends PlaneControllerClass {
       },
 
       mouseMove: (delta: Point2, position: Point2) => {
-        const tool = getVolumeTool(Store.getState().tracing).get();
-
-        if (tool === VolumeToolEnum.BRUSH) {
-          Store.dispatch(setBrushPositionAction([position.x, position.y]));
-        }
+        // TODO: when cell-hovering can be (and is) disabled via options
```
Comment (hotzenklotz) [inline app/assets/javascripts/oxalis/controller/combinations/volumetracing_plane_controller.js]: TODO?

Also, where is the option to enable / disable the effect?

File: app/assets/javascripts/oxalis/geometries/materials/plane_material_factory.js [hunk 2]
```diff
@@ -410,31 +411,47 @@ vec4 round(vec4 a) {
...
-    if (textureIdx == 0.0) {
-      return texture2D(textures[0], accessPoint).rgba;
-    } <% _.range(1, dataTextureCountPerLayer).forEach(textureIndex => { %>
-    else if (textureIdx == <%= formatNumberAsGLSLFloat(textureIndex) %>) {
-      return texture2D(textures[<%= textureIndex %>], accessPoint).rgba;
-    }
-    <% }) %>
-    else {
-      return vec4(0.5, 0.0, 0.0, 0.0);
+// Define this function for each segmentation and color layer, since iOS cannot handle
```
Comment (hotzenklotz) [inline app/assets/javascripts/oxalis/geometries/materials/plane_material_factory.js]: Does this PR also include the iOS fixes?

Comment (philippotto) [reply app/assets/javascripts/oxalis/geometries/materials/plane_material_factory.js]: I think, GH gets confused when a PR's base is another PR and does not update the diff accordingly. I switched the base to master and then back to the actual branch. Now the diff is correct and does not show the iPad stuff.

Comment (hotzenklotz) [reply app/assets/javascripts/oxalis/geometries/materials/plane_material_factory.js]: Great. I was just surprised to these changes here.

**Comment by philippotto** [comment]
> When I switch from "brush" mode to "trace" / "move" mode the "preview" of the brush shape is still active.

This issue was fixed on the base branch (which this PR was not synced with). Should work now.

> The hover effect very subtle. Sometimes I actually missed the highlighting. Perhaps we can increase the intensity of the effect a little.

You're right, I bumped up the alpha value a notch.

> Also, where is the option to enable / disable the effect?

Now it can be disabled right below the setting for segmentation opacity.

**Pull request merged**